internal/tsc_wrapped/strict_deps.ts

```diff
@@ -17,6 +17,7 @@
 
 import * as path from 'path';
 import * as ts from 'typescript';
+import {relative} from './path';
```

Comment:
again, `import * as path from 'path'`

Reply:
this is importing from the local `./path.ts` which provides versions of the four path functions which maintain backslashes. you think it would be better as `import * as path from './path'`? may be a bit confusing.

Reply:
yes, that's what I meant, that way we see the familiar `path.relative` below, and makes this delta smaller

Reply:
hmmm. this will conflict with the existing `path` imports and uses in `compiler_host.ts` and `tsconfig.ts`.

Reply:
I can just re-export the functions used out of './path'. That will be pretty clean.